code/modules/mining/materials.dm

```diff
@@ -120,6 +120,7 @@ var/global/list/initial_materials	//Stores all the matids = 0 in helping New
 	var/sheettype=null
 	var/cointype=null
 	var/value=0
+	var/alloy = 0 //Whether this material can alloy.
```

Comment:
Because there's already a system whereby any ores can be alloyed through recipes, and you're adding an extra layer of redundancy that's only really going to cause something to break in the future because people aren't necessarily going to know or care about this var.